csharp/test/Ice/namespacemd/Namespace.ice

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
The `namespacemd` test is specifically about testing `cs:namespace`, so I left it alone,
except for suppressing the deprecation warnings (I checked, and without the `suppress`, the deprecation warning was being correctly emitted).